src/reanimated2/platform-specific/webAnimations.ts

```diff
@@ -0,0 +1,278 @@
+          newTransformProp[key] = `${value}px`;
+        } else {
+          // @ts-ignore same as above.
+          newTransformProp[key] = value;
+        }
+      }
+      return newTransformProp;
+    }
+  );
+
+  for (const timestampProperties of Object.values(newAnimationData.style)) {
+    if (!timestampProperties.transform) {
+      timestampProperties.transform = newTransform;
+    } else {
+      // We insert existing transformations before ours.
+      Array.prototype.unshift.apply(
+        timestampProperties.transform,
+        newTransform
+      );
+    }
+  }
+
+  const keyframe = parseAnimationObjectToKeyframe(newAnimationData);
+
+  const styleTag = document.getElementById(
+    WEB_ANIMATIONS_ID
+  ) as HTMLStyleElement;
+
+  styleTag.sheet?.insertRule(keyframe);
+
+  return keyframeName;
+}
+
+export function generateRandomKeyframeName() {
+  const characters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
+  const length = 50;
+  let keyframeName = '';
+
+  for (let i = 0; i < length; i++) {
+    keyframeName += characters[Math.floor(Math.random() * characters.length)];
+  }
+
+  return keyframeName;
+}
+
+export function getEasingFromConfig(config: ConfigType): string {
+  if (!('easingV' in config)) {
+    return `cubic-bezier(${WebEasings.linear.toString()})`;
+  }
+
+  config = config as CustomConfig;
+
+  const easingName =
+    config.easingV !== undefined && config.easingV.name in WebEasings
+      ? config.easingV.name
+      : 'linear';
+
+  return `cubic-bezier(${WebEasings[easingName].toString()})`;
+}
+
+export function getDelayFromConfig(config: ConfigType): number {
+  const shouldRandomizeDelay = (config as CustomConfig).randomizeDelay;
+
+  const delay = shouldRandomizeDelay ? getRandomDelay() : 0;
+
+  if (!('delayV' in config)) {
+    return delay;
+  }
+
+  config = config as CustomConfig;
+
+  if (config.delayV === undefined) {
+    return delay;
+  }
+
+  return shouldRandomizeDelay
+    ? getRandomDelay((config as CustomConfig).delayV)
+    : config.delayV / 1000;
+}
+
+export function getDurationFromConfig(
+  config: ConfigType,
+  isLayoutTransition: boolean,
+  animationName: AnimationTypes
+): number {
+  const defaultDuration = isLayoutTransition
+    ? 0.3
+    : Animations[animationName].duration;
+
+  if (!('durationV' in config)) {
+    return defaultDuration;
+  }
+
+  config = config as CustomConfig;
+
+  return config.durationV !== undefined
+    ? config.durationV / 1000
+    : defaultDuration;
+}
+
+export function getCallbackFromConfig(config: ConfigType): (() => void) | null {
+  if (!('callbackV' in config)) {
+    return null;
+  }
+
+  config = config as CustomConfig;
+
+  return config.callbackV !== undefined ? config.callbackV : null;
+}
+
+export function getRandomDelay(maxDelay = 1000): number {
+  return Math.floor(Math.random() * (maxDelay + 1)) / 1000;
+}
+
+export function areDOMRectsEqual(r1: DOMRect, r2: DOMRect): boolean {
```

Comment:
```suggestion
export function areDOMRectsEqual(r1: DOMRect, r2: DOMRect) {
```

Reply:
Done in [7f8735d](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/7f8735deb90c38024a96cbe8a49260758ee8097f)